### PR TITLE
Build Page UI Change

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/flair-bi-content-header.html [hunk 1]
```diff
@@ -1,62 +1,108 @@
...
+                <!-- <div class="form-group">
+                    <label>&gt;</label>
+                </div> -->
+                <div class="form-group">
+                    <div class="dropdown">
+                        <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">
+                            <svg class="field-icon-left">
+                                <use xlink:href="#eye" />
+                            </svg>
+                            <span ng-if="vm.selectedView.viewName!==''"> {{vm.selectedView.viewName}}</span>
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi-content-header.html:46]: change the code to "!vm.selectedView.viewName"

File: src/main/webapp/app/entities/flair-bi/flair-bi-content-header.html [hunk 2]
```diff
@@ -1,62 +1,108 @@
...
+                            <span ng-if="!vm.selectedView"> Select View</span>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <div class="search-box-list inner-addon left-addon b-b-xs">
+                                <!-- <i class="glyphicon glyphicon-search"></i> -->
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi-content-header.html:51]: remove the commented  code

File: src/main/webapp/content/css/main.css [hunk 3]
```diff
@@ -4022,25 +4025,57 @@ jhi-alert .alerts .alert-danger {
     @media screen and (max-width: 576px) {
       .flairbi-content-header.get-sticky {
         left: 15px; } }
+  .flairbi-content-header .dropdown-options-v5 {
+    margin-left: 0px;
+    padding-top: 0px; }
+    .flairbi-content-header .dropdown-options-v5 .input-box {
+      display: flex;
+      border: 1px solid #e4e4e4;
```
Comment (johibkhan2) [inline src/main/webapp/content/css/main.css:4033]: try to use scss var for colours

File: src/main/webapp/content/scss/partials/_flair-bi-content-header.scss [hunk 4]
```diff
@@ -78,14 +128,14 @@
...
         .options-1::before{
-              content: "";
-              height: 35px;
-              display: block;
-              border-left: 1px lighten($icons-color, 40%) solid;
-              position: absolute;
-              left: 5%;
-              top: 50%;
-              transform: translateY(-50%);
+            //   content: "";
```
Comment (johibkhan2) [inline src/main/webapp/content/scss/partials/_flair-bi-content-header.scss:131]: remove commented code

File: src/main/webapp/content/scss/partials/_flair-bi-content-header.scss [hunk 5]
```diff
@@ -349,7 +404,7 @@
 
 .flairbi-content-header-fullscreen {
     padding: 0;
-    padding-top: 6px;
+   // padding-top: 6px;
```
Comment (johibkhan2) [inline src/main/webapp/content/scss/partials/_flair-bi-content-header.scss:407]: remove commented code

**Pull request merged**